src/map/script.c

```diff
@@ -10574,6 +10580,12 @@ static BUILDIN(guildopenstorage)
 	if (sd == NULL)
 		return true;
 
+	// Mapflag preventing from openstorage here
+	if (!pc_has_permission(sd, PC_PERM_BYPASS_NOSTORAGE) && map->list[sd->bl.m].flag.nogstorage&2) {
+		script_pushint(st, 0);
+		return false;
```

Comment:
@4144 what's your opinion on returning false?

Reply:
in some places this functions on error case already returning true, this mean probably better always return true